firebase-app-distribution/src/test/java/com/google/firebase/appdistribution/FirebaseAppDistributionTest.java

```diff
@@ -242,7 +242,7 @@ public void updateToLatestRelease_whenNewAabReleaseAvailable_showsUpdateDialog()
     AppDistributionReleaseInternal latestRelease = TEST_RELEASE_NEWER_AAB_INTERNAL.build();
     when(mockCheckForUpdateClient.checkForUpdate()).thenReturn(Tasks.forResult(latestRelease));
     firebaseAppDistribution.setCachedLatestRelease(latestRelease);
-    when(mockUpdateAppClient.updateApp(latestRelease, activity)).thenReturn(new UpdateTaskImpl());
```

Comment:
We should add a test to check the logic to only show notifications for the basic configuration